examples/pymc3_howto/lasso_block_update.ipynb

```diff
@@ -20,6 +20,10 @@
    "outputs": [],
```

Comment:
<p><code>return_inferencedata</code>  should be manually set, either to True or to False, but one of the two to avoid the warning. </p><p> </p><p>If False, an inferencedata should be created within the model context, and passed to arviz. This will 1) avoid the warning of conversion without the model context and 2) push forward arviz best practices, it is probably not too relevant here but conversion may not be cheap for some models because it requires computing all the poitnwise log likelihood values. <code>az.plot_xyz(trace)</code>  works because ArviZ internally converts the data to inferencedata, then plots.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>